packages/site/src/layouts/base.astro

```diff
@@ -22,7 +21,7 @@ const navData = searchData.map(o => pick(o, ["name", "subCategories"]))
 <html class={themes[`${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}Theme`]} lang="en">
   <head>
 		{fontHeadTags}
-    	<title>{`${title} | ${import.meta.env.SNOWPACK_PUBLIC_FRAMEWORK}.framework.dev`}</title>
+    	<title>{title}</title>
```

Comment:
Landing page is simply: `Framework.dev`.
Framework homepage titles will reflect the name of the framework as `React Framework.dev`, for example,  while
deeper routes will have: `Libraries | React Framework.dev`.